File: L4Func/js/Main.java
public class Main {
    static int num = 20;
    public static void main(String[] args) {
        // showFirstMessage("Hello World!");
        // System.out.println(num);
        System.out.printf("%d\n%d\n%d\n", calc(5, 2), calc(5, 5), calc(5, 10));
    }
    static void showFirstMessage(String text) {
        System.out.println(text);
        num = 10;
        System.out.println(num);
    }
    static int calc(int a, int b) {
        return a + b;
    }
}

File: L4Func/js/main.cpp
#include <iostream>

void showFirstMessage(std::string);
int num = 20;

int calc(int a, int b) {
    return a + b;
}

int main() {
    // showFirstMessage("Hello World!");
    // std::cout << num << "\n";
    std::cout << calc(5, 2) << "\n" << calc(5, 5) << "\n"
    << calc(5, 10) << "\n";
}

void showFirstMessage(std::string text) {
    std::cout << text << "\n";
    num = 10;
    std::cout << num << "\n";
}